static/templates/email_not_found.tpl

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
```

Comment:
Instead of creating 2 files(static/templates/email_not_found.tpl, static/templates/email_not_found_staging.tpl), keep just one file & use `config`(see next comment!)